pkg/ocmagenthandler/ocmagenthandler_deployment.go

```diff
@@ -28,43 +28,71 @@ func buildOCMAgentDeployment(ocmAgent ocmagentv1alpha1.OcmAgent) appsv1.Deployme
 		MatchLabels: labels,
 	}
 
-	var volumes []corev1.Volume
-	var volumeMounts []corev1.VolumeMount
-	// Define a volume/volume mount for the access token secret
-	var secretVolumeSourceDefaultMode int32 = 0600
+	// Define a volumes for the config
 	tokenSecretVolumeName := ocmAgent.Spec.TokenSecret
-	volumes = append(volumes, corev1.Volume{
-		Name: tokenSecretVolumeName,
-		VolumeSource: corev1.VolumeSource{
-			Secret: &corev1.SecretVolumeSource{
-				SecretName:  ocmAgent.Spec.TokenSecret,
-				DefaultMode: &secretVolumeSourceDefaultMode,
-			},
-		},
-	})
-	volumeMounts = append(volumeMounts, corev1.VolumeMount{
-		Name:      tokenSecretVolumeName,
-		MountPath: filepath.Join(oah.OCMAgentSecretMountPath, tokenSecretVolumeName),
-	})
-
-	// Define a volume/volume mount for the config
 	configVolumeName := ocmAgent.Spec.OcmAgentConfig
+	trustedCaVolumeName := oah.TrustedCaBundleConfigMapName
+	var secretVolumeSourceDefaultMode int32 = 0600
 	var configVolumeSourceDefaultMode int32 = 0600
-	volumes = append(volumes, corev1.Volume{
-		Name: configVolumeName,
-		VolumeSource: corev1.VolumeSource{
-			ConfigMap: &corev1.ConfigMapVolumeSource{
-				LocalObjectReference: corev1.LocalObjectReference{
-					Name: ocmAgent.Spec.OcmAgentConfig,
+	var trustedCaVolumeSourceDefaultMode int32 = 0420
```

Comment:
This ends up being converted to decimal, which becomes `272` as the `defaultMode` value in the actual deployment created - which then translates to `0420` in octal form as the actual file permission being mounted.

(From my testing of the PR below)
```bash
      - configMap:
          defaultMode: 272
          items:
          - key: ca-bundle.crt
            path: tls-ca-bundle.pem
          name: trusted-ca-bundle
        name: trusted-ca-bundle
```
(other operators will have that value reporting as `420`)

So instead in the code I think this needs to change to the octal equivalent we want, `0644`, and that will result in it having a default mode of `420` (decimal form) in the deployment like the other operators do.

Reply:
```suggestion
	var trustedCaVolumeSourceDefaultMode int32 = 0644
```